admob/README.md

```diff
@@ -0,0 +1,61 @@
+AdMob by Google Quickstart
+=======================
+
+The AdMob by Google Flutter quickstart demonstrates how to display an interstitial ad and
+a banner ad.  AdRequest and AdView are used to display a banner ad
+and InterstitialAd is used to display the interstitial ad.
+
+Introduction
+------------
+
+- [Read more about AdMob by Google](https://firebase.google.com/docs/admob/)
+
+Getting Started
+---------------
+
+## Android
+
+- [Add AdMob to your Flutter Project](https://developers.google.com/admob/flutter/quick-start#android).
+- Configure your AdMob app id:
+  - In `android/app/src/main/AndroidManifest.xml` change the `ca-app-pub-xxxxxxxxxxxxxxxx~yyyyyyyyyy` value to your AdMob app id.
+- Run the sample on your Android device or emulator.
+- The running sample displays a test banner ad and a test interstitial add.
+
+## iOS
+
+- [Add AdMob to your Flutter Project](https://developers.google.com/admob/flutter/quick-start#ios).
+- Configure your AdMob app id:
+  - In `ios/Runner/Info.plist` change the `ca-app-pub-xxxxxxxxxxxxxxxx~yyyyyyyyyy` value to your AdMob app id.
+- Run the sample on your iOS device or simulator.
+- The running sample displays a test banner ad and a test interstitial add.
```

Comment:
1. nit: s/interstitial add./interstitial ad.
2. Do you have to click the button to show the interstitial ad? If so please add that as a step here.

Reply:
Fixed :)